cmd/test_load.go

```diff
@@ -240,14 +241,23 @@ func loadAndConfigureTest(
 	return test.consolidateDeriveAndValidateConfig(gs, cmd, cliConfigGetter)
 }
 
+func loadSystemCertPool(gs *globalState) {
+	if _, err := x509.SystemCertPool(); err != nil {
+		gs.logger.WithError(err).Warning("Unable to load system cert pool")
+	}
+}
+
 func (lct *loadedAndConfiguredTest) buildTestRunState(
+	gs *globalState,
 	configToReinject lib.Options,
 ) (*lib.TestRunState, error) {
 	// This might be the full derived or just the consodlidated options
 	if err := lct.initRunner.SetOptions(configToReinject); err != nil {
 		return nil, err
 	}
 
+	go loadSystemCertPool(gs)
```

Comment:
```suggestion
    // it pre-loads the system certificates so it can be avoided doing it on the first TLS request.
    // It loads in async so we can avoid blocking the rest of the loading process,
    // because the test will not be stopped if it fails.
	go loadSystemCertPool(gs.logger)
```
I guess something like this explaining why could be helpful. Feel free to add more context if it isn't clear enough.